credential-exchange-types/src/format.rs

```diff
@@ -83,7 +83,7 @@ pub struct Item {
     pub extensions: Option<Vec<Extension>>, // default []
 }
 
-#[derive(Debug, Serialize, Deserialize)]
+#[derive(Debug, Serialize, Deserialize, PartialEq)]
```

Comment:
**Suggestion:** Should add `Eq` as well according to [clippy](https://rust-lang.github.io/rust-clippy/master/index.html#derive_partial_eq_without_eq)

Reply:
I need to get around to configuring clippy to warn on those things.